app/models/person_with_remote_fields.rb

```diff
@@ -61,7 +61,8 @@ def remote_attributes
 
   def location_params
     if location && (location.changed & Location::ADDRESS_FIELDS.map(&:to_s)).any?
-      location.as_json  # maybe dangerous to reuse serializable_hash here?
+      location.as_json(methods: [:state_abbrev],
```

Comment:
This is duplicating the default behavior of `Location#serializable_hash` so that the default behavior can change without affecting this method. Not sure if that's really necessary. Maybe we can just rely on the spec?